### PR TITLE
vaunt: update achievement images and add vaunt documentation

### PR DESCRIPTION
**Pull request by jeff1010322**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Updated the 1st_place.png and contributor.png Vaunt achievement images.
Added a README.md to the .vaunt directory describing how users can update their profile to show their badges.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by rpaik** [comment]
Thanks @jeff1010322 

I wonder if it'd be better to have a more descriptive name for the readme file. Maybe something like `achievement-badges-readme.md` @qiancai, what do you think?

**Comment by qiancai** [comment]
> Thanks @jeff1010322
> 
> I wonder if it'd be better to have a more descriptive name for the readme file. Maybe something like `achievement-badges-readme.md` @qiancai, what do you think?

Yes, just using "readme" is a little confusing.  how about something like `add-docs-dash-badge.md`?

**Comment by rpaik** [comment]
> > Thanks @jeff1010322
> > I wonder if it'd be better to have a more descriptive name for the readme file. Maybe something like `achievement-badges-readme.md` @qiancai, what do you think?
> 
> Yes, just using "readme" is a little confusing. how about something like `add-tidb-docs-dash-badge.md`?

This works for me. 

@jeff1010322 could you update the file name?

**Comment by jeff1010322** [comment]
> > > Thanks @jeff1010322
> > > I wonder if it'd be better to have a more descriptive name for the readme file. Maybe something like `achievement-badges-readme.md` @qiancai, what do you think?
> > 
> > 
> > Yes, just using "readme" is a little confusing. how about something like `add-tidb-docs-dash-badge.md`?
> 
> This works for me.
> 
> @jeff1010322 could you update the file name?

Yes, I have just updated the file name.

**Review by qiancai — COMMENTED** [review]
Hi @rpaik and @jeff1010322, I've added comments to make instructions shorter so contributors can get started quickly.  About the parts of integrating the Vaunt Developer Card and Community Boards, I think it is okay to keep them in this file so anyone interested in that can have a try, but perhaps it is better to only share the key info there so this file won't be too long. PTAL. Thanks.

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    </p>
+    ```
+
+ Your GitHub profile now displays your TiDB Docs Dash badge. For more information about managing your profile README, see [GitHub docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme).
+
+## (Optional) Explore more with Vaunt
+
+TiDB Docs Dash badges are granted via [Vaunt](https://vaunt.dev/), a developer relations platform that aims to empower and grow developer communities.
+
+You can also use Vaunt to showcase your contributions to open-source projects as follows:
```
Comment (qiancai) [inline .vaunt/add-tidb-docs-dash-badge.md:37]: @jeff1010322, I am not sure whether the Vaunt app is a prerequisite for integrating the developer card and community boards. Would you please help confirm? If yes, we also need to indicate that information here.

Comment (rpaik) [reply .vaunt/add-tidb-docs-dash-badge.md:37]: Maybe we can rephrase this line 37 to something like "You can use additional Vaunt features to showcase your contributions to other open source projects as follows:"?

Comment (jeff1010322) [reply .vaunt/add-tidb-docs-dash-badge.md]: I have updated this line as suggested.

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# How to Add TiDB Docs Dash Badges to your GitHub Profile
```
Comment (hfxsd) [inline .vaunt/add-tidb-docs-dash-badge.md:1]:
```suggestion
# How to Add TiDB Docs Dash Badges to Your GitHub Profile
```

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+If you have already set up your GitHub profile README, skip the following and go to [Step 2. Edit your profile README](#step-2-edit-your-profile-readme).
+
+1. In the upper-right corner of any GitHub page, click **+**, and then click **New repository**.
+
+    <img src="https://docs.github.com/assets/cb-34248/mw-1440/images/help/repository/repo-create-global-nav-update.webp" width="350" />
+
+2. Type a repository name that matches your GitHub username. For example, if your username is `ilovetidb`, the repository name must be `ilovetidb`.
+3. Choose **Public** for the repository visibility.
+4. Select **Initialize this repository with a README file**.
```
Comment (hfxsd) [inline .vaunt/add-tidb-docs-dash-badge.md:17]:
```suggestion
4. Under **Initialize this repository with:**, select **Add a README file**.
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**